applications/DEMApplication/python_scripts/DEM_analysis_stage.py

```diff
@@ -330,71 +331,98 @@ def GetProblemNameWithPath(self):
         return os.path.join(self.mdpas_folder_path, self.DEM_parameters["problem_name"].GetString())
 
     def GetMpFilename(self):
-        return self.GetProblemNameWithPath() + "DEM"
+        return 'DEM'
 
     def GetInletFilename(self):
-        return self.GetProblemNameWithPath() + "DEM_Inlet"
+        return 'DEM_Inlet'
 
     def GetFemFilename(self):
-        return self.GetProblemNameWithPath() + "DEM_FEM_boundary"
+        return 'DEM_FEM_boundary'
 
     def GetClusterFilename(self):
-        return self.GetProblemNameWithPath() + "DEM_Clusters"
+        return 'DEM_Clusters'
+
+    def GetInputFilePath(self, file_name):
+        return self.GetProblemNameWithPath() + file_name
 
     def GetProblemTypeFilename(self):
         return self.DEM_parameters["problem_name"].GetString()
 
-    def ReadModelParts(self, max_node_Id=0, max_elem_Id=0, max_cond_Id=0):
-        spheres_mp_filename = self.GetMpFilename()
-        model_part_io_spheres = self.model_part_reader(spheres_mp_filename, max_node_Id, max_elem_Id, max_cond_Id)
+    def ReadModelPartsFromRestartFile(self, model_part_import_settings):
+        Logger.PrintInfo('DEM', 'Loading model parts from restart file...')
+        from DEM_restart_utility import DEMRestartUtility
```

Comment:
imprt should be at the top of the file and with absolute path

Reply:
@philbucher 

There are more of these imports, but this should be cleaned all over DEMApplication in a future PR